srcpkgs/wayfire/INSTALL.msg

```diff
@@ -0,0 +1,6 @@
+Before running Wayfire, copy the default configuration file to your user:
+
+  cp /usr/share/wayfire/wayfire.ini.default ~/.config/wayfire.ini
+
+You also probably want to install elogind and dbus-elogind, then restart your computer
```

Comment:
I don't think this needs dbus-elogind

Reply:
Without elogind, it depends on systemd-login.

Reply:
I said dbus-elogind. That is only related to specific dbus feature that as far as I can tell is only used by bluez.

Reply:
removed dbus-elogind, I had assumed wrong :(